src/rj_common/CMakeLists.txt

```diff
@@ -23,6 +23,19 @@ find_package(rj_param_utils REQUIRED)
 find_package(rj_utils REQUIRED)
 find_package(rj_drawing_msgs REQUIRED)
 
+set(RJ_COMMON_DEPS
+  rj_msgs
+  Eigen3
+  rclcpp
+  rclcpp_components
+  rj_convert
+  rj_protos
+  rj_geometry
+  rj_param_utils
+  rj_utils
+  rj_drawing_msgs
+)
```

Comment:
IIRC rj_common doesn't use these dependencies so they should not be listed in CMakeLists.txt.  Additionally, if they were used here they would need to be declared in find_package and the package.xml file for rj_common